docs/content/operations/dashboard.md

```diff
@@ -104,8 +104,44 @@ rule = "Host(`traefik.example.com`) && (PathPrefix(`/api`) || PathPrefix(`/dashb
 ??? example "Dashboard Dynamic Configuration Examples"
     --8<-- "content/operations/include-dashboard-examples.md"
 
+### Custom API Base Path
+
+As shown above, by default Traefik exposes its API and Dashboard under the `/` base path,
+which means that respectively the API is served under the `/api` path,
+and the dashboard under the `/dashboard` path.
+
+However, it is possible to configure this base path:
```

Comment:
```suggestion
It is possible to configure this base path:
```

Reply:
@mloiseleur I'm not sure about this suggestion, why suppressing this adverb?